runtime/amplitude/src/lib.rs

```diff
@@ -85,11 +85,13 @@ pub use sp_runtime::BuildStorage;
 pub use dia_oracle::dia::AssetId;
 pub use issue::{Event as IssueEvent, IssueRequest};
 pub use nomination::Event as NominationEvent;
+#[allow(unused_imports)]
```

Comment:
Why do we need to keep this import?

Reply:
I separated and added the tag to remove the unused warning when we compile with with the `runtime-benchmarks`, because it is not [used](https://github.com/pendulum-chain/pendulum/blob/12d19aa4ead6aaa8a42530b3b47f0787d90d5b78/runtime/amplitude/src/lib.rs#L275) in that case.